h/routes.py

```diff
@@ -189,6 +189,7 @@ def includeme(config):
 
     # Client
     config.add_route("sidebar_app", "/app.html")
+    config.add_route("notebook_app", "/notebook.html")
```

Comment:
I would suggest to use just `/notebook` for consistency with the majority of other routes, including the other client routes (`/stream`, `/a/<annotation_id>`). The `app.html` path is just legacy at this point which we haven't had to worry about too much because users don't see it.

Reply:
OK, yeah, I heartily agree with this. The `html` was awkward. Roger wilco.